### PR TITLE
[T3A2][W15-A2] Wang Ce

### PR DESCRIPTION
**Pull request by wangce94**

### CODE REVIEW

**Comment by wangce94** [comment]
Ready for review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -32,5 +32,9 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    public String getFeedback() {
```
Comment (mauris) [inline src/seedu/addressbook/commands/CommandResult.java:36]: would be good to include method header comments

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedback(), DIVIDER);
```
Comment (mauris) [inline src/seedu/addressbook/ui/TextUi.java:128]: good that you update the other references

**Comment by mauris** [comment]
Some comments added. Feel free to add more comment if you have any questions. Once you are done, you may close the PR after reading comments.

**Pull request closed without merging**